tools/ubuntu-setup/pip.sh

```diff
@@ -1,4 +1,3 @@
 sudo apt-get install -y python-pip
 sudo pip install jsonschema
 sudo pip install argcomplete
-sudo pip install docker-py
```

Comment:
The dependency only strikes on usage of ansible, so the order doesn't matter here i suppose.